.env.sample

```diff
@@ -5,3 +5,6 @@
 
 # Google Analytics Tracking Code
 #GA_TRACKING_ID=
+
+# Fully qualified site URL, without trailing slash
+#SITE_URL=http://localhost:3000
```

Comment:
Maybe we should actually consider this as the default value, so I would say to uncomment it.